boot.js

```diff
@@ -467,8 +467,8 @@ function timeoutCall (func, rootErr, context, cb) {
   }
 }
 
-function closeWithCbOrNextTick (func, cb, context) {
-  context = this._server
+function closeWithCbOrNextTick (func, cb) {
```

Comment:
this is never called with the 3rd argument?

Reply:
never called with 3rd argument